src/App.js

```diff
@@ -1,14 +1,78 @@
-import React from 'react';
+import React from 'react'
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './components/TodoComponents/Todo.css'
+
+
+localStorage.getItem('state')
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props){
+    super(props);
+    this.state =
+      {Todos: 
+        [
+          {task: 'watch this week\'s The Orville',
```

Comment:
generally I would recommend separating your list of static tasks from your the array of tasks you plan to add to make sure they don't ever clash with each other. Though I don't think it should mess anything up in this project.